### ISSUE TITLE
Stop method doesn't reset the node to its default position

### ISSUE THREAD

**Issue by averroes96** [Typhon0/AnimateFX#13]
### Description

I'm trying to animate a field using methods `onKeyPressed`, `onKeyTyped` and `onKeyReleased` so that whenever the entered value is invalid or wrongly typed, it basically starts an animation indicating that you're making an error, but once the entered value is valid i basically call the `stop` method to end the animation but sometimes it stops the field in a different position than the his initial postion. i looked if there's another method that do reset the field to its initial postion but found none.

**Comment by Typhon0** [comment]
Hi, you can use this function : `setResetOnFinished(true)`.
It will reset the node position when it's finished or when you call `stop()`

**Comment by averroes96** [comment]
I basically tried this one earlier, just after instanciating the animation :

```java
AnimateFX animField = new Shake(field);
animField.setResetOnFinished(true)
```

Still didn't work.

**Comment by Typhon0** [comment]
Fixed with commit [8e899e1](https://github.com/Typhon0/AnimateFX/commit/8e899e11e8371f2e3e79f77126e7338d2c32e38a)

**Comment by Typhon0** [comment]
Fixed with commit [8e899e1](https://github.com/Typhon0/AnimateFX/commit/8e899e11e8371f2e3e79f77126e7338d2c32e38a)
